labs/in-dev-container/deploy/ngsa-memory.yaml

```diff
@@ -0,0 +1,125 @@
+apiVersion: apps/v1
```

Comment:
Looks like this file is identical to `labs/in-cluster/deploy/ngsa-memory.yaml`. Is that intentional or can they be consolidated in one place?

Reply:
the initial idea was to have labs independent from each other.